lib.php

```diff
@@ -21,6 +21,18 @@
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
+defined('MOODLE_INTERNAL') || die();
+
 function tool_heartbeat_status_checks() {
-    return [new \tool_heartbeat\check\authcheck()];
+    return [
+        new \tool_heartbeat\check\authcheck(),
+        new \tool_heartbeat\check\logcheck(),
+    ];
+}
+
+function tool_heartbeat_after_config() {
+    stream_wrapper_register('syslog', 'tool_heartbeat\wrapper\syslog')
```

Comment:
this doesn't even install:

Error
Exception - syntax error, unexpected 'throw' (T_THROW)
Debug info:
Error code: generalexceptionmessage
Stack trace:
line 35 of /admin/tool/heartbeat/lib.php: ParseError thrown
line 7833 of /lib/moodlelib.php: call to core_component::get_plugin_list_with_file()
line 1084 of /lib/setup.php: call to get_plugins_with_function()
line 651 of /config.php: call to require_once()
line 88 of /admin/index.php: call to require()